.github/workflows/ci.yml

```diff
@@ -5,16 +5,36 @@ on:
       - master
   pull_request:
 jobs:
+  ubuntuUnitWithJDK8And11:
+    name: ubuntu-latest tests ${{ matrix.java-version }}
```

Comment:
Instead of duplicating the unit tests for ubuntu, could we try to use `exclude` instead? 🤔 

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

```yml
    os: [macos, windows, ubuntu]
    java: [8, 11]
    exclude:
      - os: windows
        java: 8
      - os: macos
        java: 8
```

Reply:
Sure. Updated.